Lumberjack.xcodeproj/project.pbxproj

```diff
@@ -95,77 +95,77 @@
-				620EEE7F1BFA65CE00D1B9CB /* DDContextFilterLogFormatter.h in CopyFiles */,
-				620EEE801BFA65CE00D1B9CB /* DDDispatchQueueLogFormatter.h in CopyFiles */,
-				620EEE811BFA65CE00D1B9CB /* DDMultiFormatter.h in CopyFiles */,
+				0A26B5BB22DE37AA004EE6A7 /* DDFileLogger+Internal.h in CopyFiles */,
+				0A26B5A522DE3503004EE6A7 /* CocoaLumberjack.h in CopyFiles */,
+				0A26B59422DE34E9004EE6A7 /* DDLog.h in CopyFiles */,
+				0A26B59622DE34E9004EE6A7 /* DDMultiFormatter.h in CopyFiles */,
+				0A26B59722DE34E9004EE6A7 /* DDAbstractDatabaseLogger.h in CopyFiles */,
+				0A26B59822DE34E9004EE6A7 /* DDLoggerNames.h in CopyFiles */,
+				0A26B59922DE34E9004EE6A7 /* DDLogMacros.h in CopyFiles */,
+				0A26B59A22DE34E9004EE6A7 /* DDASLLogCapture.h in CopyFiles */,
+				0A26B59B22DE34E9004EE6A7 /* DDFileLogger.h in CopyFiles */,
+				0A26B59C22DE34E9004EE6A7 /* DDContextFilterLogFormatter.h in CopyFiles */,
+				0A26B59D22DE34E9004EE6A7 /* DDTTYLogger.h in CopyFiles */,
+				0A26B59E22DE34E9004EE6A7 /* DDAssertMacros.h in CopyFiles */,
+				0A26B59F22DE34E9004EE6A7 /* DDFileLogger+Buffering.h in CopyFiles */,
+				0A26B5A022DE34E9004EE6A7 /* DDDispatchQueueLogFormatter.h in CopyFiles */,
+				0A26B5A122DE34E9004EE6A7 /* DDOSLogger.h in CopyFiles */,
+				0A26B5A222DE34E9004EE6A7 /* DDLog+LOGV.h in CopyFiles */,
+				0A26B5A322DE34E9004EE6A7 /* CLIColor.h in CopyFiles */,
+				0A26B5A422DE34E9004EE6A7 /* DDASLLogger.h in CopyFiles */,
+				0752523F22E07666005D30B3 /* DDLegacyMacros.h in CopyFiles */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
-		07305D7A216790F300C61363 /* SwiftLogLevel.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = SwiftLogLevel.h; sourceTree = "<group>"; };
-		0A3F9CE921E3F8AF00F3B707 /* DDFileLogger+Internal.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "DDFileLogger+Internal.h"; sourceTree = "<group>"; };
-		0AA59E1721DD2A080031787C /* DDFileLogger+Buffering.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "DDFileLogger+Buffering.h"; sourceTree = "<group>"; };
-		0AA59E1821DD2A080031787C /* DDFileLogger+Buffering.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = "DDFileLogger+Buffering.m"; sourceTree = "<group>"; };
-		0AE6D52321941F0900B2A35D /* DDLoggerNames.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = DDLoggerNames.m; sourceTree = "<group>"; };
-		0AE6D52421941F0900B2A35D /* DDLoggerNames.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = DDLoggerNames.h; sourceTree = "<group>"; };
+		0752523B22E075E1005D30B3 /* SwiftLogLevel.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; name = SwiftLogLevel.h; path = Sources/CocoaLumberjackSwiftSupport/include/SwiftLogLevel.h; sourceTree = SOURCE_ROOT; };
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/CocoaLumberjack/CocoaLumberjack/blob/40371aaae76c6a0e7194a8b50db3f6cd0718d85a/Lumberjack.xcodeproj/project.pbxproj#L123"></span>Files should be in sync with project structure</td>
    </tr>
  </tbody>
</table>